api/user/user.controller.js

```diff
@@ -60,4 +60,10 @@ export async function updateUserHandler(req, res) {
   }
 }
 
-export function deleteUserHandler(req, res) {}
+function deleteUserHandler(req, res) {}
+
+exports.getAllUserHandler = getAllUserHandler;
+exports.getSingleUserHandler = getSingleUserHandler;
+exports.createUserHandler = createUserHandler;
+exports.updateUserHandler = updateUserHandler;
+exports.deleteUserHandler = deleteUserHandler;
```

Comment:
This can be changed to something like this

```suggestion
module.exports = {
  getAllUserHandler,
  getSingleUserHandler,
  createUserHandler,
  updateUserHandler,
  deleteUserHandler,
}
```